### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

File: tests/Fixer/PhpTag/NoClosingTagFixerTest.php [hunk 1]
```diff
@@ -121,9 +121,12 @@ function bar()
                 '<?php ?>',
             ),
             'No code, only comment' => array(
-                '<?php /* license */ ',
+                '<?php /* license */',
```
Comment (SpacePossum) [inline tests/Fixer/PhpTag/NoClosingTagFixerTest.php]: not sure about the fixer removing the white space, WDYT @keradus ?

Comment (keradus) [inline tests/Fixer/PhpTag/NoClosingTagFixerTest.php]: why should it? i don't think it should.

Comment (SpacePossum) [inline tests/Fixer/PhpTag/NoClosingTagFixerTest.php]: hmm not sure why this was done in the first place, it seems to be doing that for all cases

Comment (keradus) [inline tests/Fixer/PhpTag/NoClosingTagFixerTest.php]: very well...
then commit log will be directly passed into project branch log. please adjust msg-log to be meaningful

**Comment by keradus** [comment]
then commit log will be directly passed into project branch log. please adjust msg-log to be meaningful

**Comment by SpacePossum** [comment]
will do :)  feel free to make a patch of the commit apply directly to your branch and squash it, you know I do not care about authorship

**Comment by SpacePossum** [comment]
added more tests
ref.: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2587

**Comment by keradus** [comment]
thanks ;)

**Pull request merged**